api/.strapi-updater.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
can you add this file to the gitignore :)

Reply:
Done